src/main/resources/static/allCards.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All cards</title>
+    <link rel="stylesheet" href="/public/allCards.css">
+</head>
+<body>
+<div id="cardsContainer">
+    <button id="backButton" onclick="goBack()">Back</button>
+    <h1 id="allCardsHeader">All cards</h1>
+    <div id="cardsList"></div>
+</div>
+
+
+<script>
+    let cardsArray = [];
+
+    function drawCardList(cardsArray){
+        let cardsList = document.getElementById("cardsList");
+        let cards = "";
+        cardsArray.forEach((card, index) => {
+            cards += `
+            <div class="cardContainer">
+                <span>Card Number:${card.cardNumber}</span>
+                <br>
+                <span>Card Holder: ${card.holderName}</span>
+                <br>
+                <button onclick="removeCard('${card.cardNumber}', ${index})">Remove</button>
+                <button onclick="updateCard('${card.cardNumber}')">Update</button>
+            </div>`
+        });
+        cardsList.innerHTML = cards;
+    }
+    function fetchData() {
+        fetch("/card/getAllCards", {
+            method: "GET"
+        })
+            .then(response => response.json())
+            .then(response => { cardsArray = response; drawCardList(cardsArray)})
```

Comment:
You could call `drawCardList(response)` directly.

Reply:
I'm keeping the response in cardsArray variable in order to remove cards from the same array when I get the response back from removeCard function. I also thought about reloading the whole list from the server instead after removing a card, redrawing everything